### PR TITLE
SeyLII

### PR DESCRIPTION
**Pull request by nickmwangemi**
image

image

image

image

### CODE REVIEW

File: seylii/templates/liiweb/home.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    {% include 'peachjam/_recent_document_list.html' with documents=recent_judgments view_more_text=text view_more_url_name='judgment_list' %}
+  </section>
+  <section class="pb-3 pt-3">
+    <h2 class="mb-4 heading-underlined">{% trans 'Recent Legislation' %}</h2>
+    {% trans 'View more legislation' as text %}
+    {% include 'peachjam/_recent_document_list.html' with documents=recent_legislation view_more_text=text view_more_url_name='legislation_list' %}
+  </section>
+{% endblock %}
+{% block second-column-content %}
+  <div>
```
Comment (actlikewill) [inline seylii/templates/liiweb/home.html:41]: @nickmwangemi The pull requests I've just merges pulls this section into the base template, you don't need to add the social media tabs now. There are some blocks that you can reuse. Rebase the branch and check them out.

File: seylii/templates/peachjam/_header.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{% extends 'peachjam/_header.html' %}
+{% load static i18n %}
+{% block top-bar %}{% endblock %}
+{% block navbar-logo %}{{ APP_NAME }}{% endblock %}
```
Comment (actlikewill) [inline seylii/templates/peachjam/_header.html:4]: @nickmwangemi You can omit this

File: seylii/templates/peachjam/_set_language.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<!-- djlint:off -->
```
Comment (actlikewill) [inline seylii/templates/peachjam/_set_language.html:1]: @nickmwangemi whats this file for?

Comment (nickmwangemi) [reply seylii/templates/peachjam/_set_language.html:1]: @actlikewill Removes the language selector

**Pull request merged**